packages/asset-swapper/src/constants.ts

```diff
@@ -49,6 +49,7 @@ const DEFAULT_SWAP_QUOTER_OPTS: SwapQuoterOpts = {
         makerAssetOfferings: {},
         skipBuyRequests: false,
     },
+    samplerCallDataOverrides: {},
```

Comment:
I do have a slight concern about how this will impact ganache tests if `overrides` isn't supported? I'm not entirely convinced of having it as the default just yet.

Reply:
Can we just explicitly set this to `{}` in tests?

Reply:
sounds reasonable